scripts/create_translation_file.php

```diff
@@ -74,8 +81,11 @@
 			foreach( $i18n_instances as $instance )
 			{
 				if( !translation_exits( $instance[ 'context' ], $instance[ 'source' ] ) )
-				{
-					$result[ $instance[ 'context' ] ][ md5( $instance[ 'source' ] ) ] = $instance;
+				{                    
+                    if( in_array( $instance[ 'context' ], $contexts, false ) || $contexts[ 0 ] == "" )
```

Comment:
replace this line with:
if( empty( $contexts ) || in_array( $instance[ 'context' ], $contexts, false ) )